src/coreclr/nativeaot/docs/android-bionic.md

```diff
@@ -6,11 +6,9 @@ Not a full Android experience is available - it's only possible to publish for t
 
 The minimum API level is 21 at the time of writing the document, but search for AndroidApiLevelMin in this repo for more up-to-date information.
 
-NOTE: There's an existing issue that puts the minimum Android version where things work to Android 10: https://github.com/dotnet/runtime/issues/92196
-
 To build for Bionic:
 
-* Ensure you have [Android NDK](https://developer.android.com/ndk/downloads) for your system downloaded and extracted somewhere. We build and test with NDK r23c but anything newer should also work. Double check with the NDK version referenced [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/libraries/testing-android.md), which might be more up-to-date than this document.
+* Ensure you have [Android NDK](https://developer.android.com/ndk/downloads) for your system downloaded and extracted somewhere. We build and test with NDK r23c but anything newer should also work. Newer releases of the NDK might require a workaround (https://github.com/dotnet/runtime/issues/92272). Double check with the NDK version referenced [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/libraries/testing-android.md), which might be more up-to-date than this document.
```

Comment:
People going through these steps won't be able to tell whether they need the workaround or not at this point. Instead of this, we may want to mention this after the `publish` step. Something like:

If you see `error : version script assignment of 'V1.0' to symbol '_init' failed: symbol not defined` error, add these lines to your .csproj to workaround [a known issue]((https://github.com/dotnet/runtime/issues/92272) with certain NDK versions:
```
<ItemGroup Condition="'$(RuntimeIdentifier)' == 'linux-bionic'">
  <LinkerArg Include="-Wl,--defsym,_init=__libc_init" />
  <LinkerArg Include="-Wl,--defsym,_fini=__libc_fini" />
</ItemGroup
```

Reply:
Could the workaround be integrated into the SDK so that it's applied automatically?

Reply:
> Could the workaround be integrated into the SDK so that it's applied automatically?

Yes, we can backport the fix once the issue is fixed...